src/OctoshiftCLI.Tests/Octoshift/Services/GithubApiTests.cs

```diff
@@ -3368,12 +3368,54 @@ public async Task AbortMigration_Returns_True_On_Success()
             .ReturnsAsync(response);
 
         // Act
-        var expectedBooleanResponse = await _githubApi.AbortMigration(migrationSourceId);
+        var expectedBooleanResponse = await _githubApi.AbortMigration(migrationId);
 
         // Assert
         expectedBooleanResponse.Should().Be(actualBooleanResponse);
     }
 
+    [Fact]
+    public async Task AbortMigration_Surfaces_Error_With_Incorrect_Migration_ID()
+    {
+        // Arrange
+        const string migrationId = "1234";
+
+        const string query = @"
+                mutation abortRepositoryMigration(
+                    $migrationId: ID!,
+                )";
+        const string gql = @"
+                abortRepositoryMigration(
+                    input: { 
+                        migrationId: $migrationId
+                    })
+                   { success }";
+
+        var payload = new
+        {
+            query = $"{query} {{ {gql} }}",
+            variables = new
+            {
+                migrationId,
+            },
+            operationName = "abortRepositoryMigration"
+        };
+
+        const string expectedErrorMessage = $"Invalid migration id: {migrationId}";
+
+        // Act
+        _githubClientMock
+            .Setup(m => m.PostGraphQLAsync(It.IsAny<string>(), It.IsAny<object>(), null))
+            .ThrowsAsync(new OctoshiftCliException("Could not resolve to a node"));
+
+        // Assert
+
```

Comment:
the `_githubClientMock` is technically still part of the arrange, in this case act and assert are both happening in the invoke like:
```suggestion
        _githubClientMock
            .Setup(m => m.PostGraphQLAsync(It.IsAny<string>(), It.IsAny<object>(), null))
            .ThrowsAsync(new OctoshiftCliException("Could not resolve to a node"));

        // Act, Assert
```